Test.Activities.Automation.WCFService/ISAPI/AutomationService.svc.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.ServiceModel.Activation;
+using Test.Activities.Automation.ActivityLib.Models;
+using Test.Activities.Automation.ActivityLib.Models.Activity.Services.SyncActivityService;
+
+namespace Test.Activities.Automation.WCFService
+{
+    [AspNetCompatibilityRequirements(RequirementsMode = AspNetCompatibilityRequirementsMode.Required)]
+    public class AutomationService : IAutomationService
+    {
+        private ILogger _logger;
+
+        public AutomationService()
+        {
+            try
+            {
+                _logger = new ULSLogger(GetType().FullName);
+            }
+            catch
+            {
+            }
+        }
+
+        public HttpStatusCode FillActivities(IEnumerable<ActivityInfo> activities)
```

Comment:
I agree with you. Earlier I returned true/false from this method. But then I decided to change true/false to status code and forgot to put it in the head instead of the body.